docs/api/document.rst

```diff
@@ -518,6 +518,13 @@ This class represents a document. It can be constructed from a file or from memo
 
       .. note:: In a typical use case, a page :ref:`Pixmap` should be taken after annotations / widgets have been added or changed. To force all those changes being reflected in the page structure, this method re-instates a fresh copy while keeping the object hierarchy "document -> page -> annotations/widgets" intact.
 
+   .. method:: Recolor(int pageNum, int colorNum)
+   .. method:: Recolor(int pageNum, string colorSpaceName)
```

Comment:
You need a carriage return after the method definition otherwise the next line looks like another method ( see screenshot )
<img width="795" alt="Screenshot 2024-10-07 at 12 37 42" src="https://github.com/user-attachments/assets/386d05ab-7778-484c-bbcb-6280d25589de">